src/traces/pie/attributes.js

```diff
@@ -18,7 +18,7 @@ var extendFlat = require('../../lib/extend').extendFlat;
 var textFontAttrs = fontAttrs({
     editType: 'calc',
     arrayOk: true,
-    colorEditType: 'style',
+    colorEditType: 'plot',
```

Comment:
Sure, would be nice someday to get `style` to work but this isn't a real high-traffic pathway (nor a heavy plot type, unless someone's really abusing it!)